worlds/stardew_valley/data/skill.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass, field
+
+from ..data.game_item import kw_only
+
+
+@dataclass(frozen=True)
+class Skill:
+    name: str
+    has_mastery: bool = field(**kw_only)
+
+    @property
+    def max_level(self):
```

Comment:
What is this for, and why would skills go to 15 with masteries?

Reply:
My bad, I misunderstood the wiki 😅

Reply:
Removed.